### ISSUE TITLE
Making the default settings closer to stock Pure

### ISSUE THREAD

**Issue by Calinou** [nickcox/pure-pwsh#3]
I think it would make sense for this port to use the same settings as stock Pure for consistency. This way, users switching between zsh and PowerShell can benefit from the same appearance out of the box (I use PowerShell Core on Linux occasionally).

For the record, I use those settings to make pure-pwsh look closer to stock Pure:

```powershell
$esc = [char]27
$pure.PwdColor = "$esc[34m"
$pure.BranchColor = "$esc[90m"
$pure.RemoteColor = "$esc[36m"
$pure.PromptColor = "$esc[35m"
```

![zsh_pwsh_pure](https://user-images.githubusercontent.com/180032/49401100-77886900-f746-11e8-9d39-6cbd4015c8a1.png)

**Comment by nickcox** [comment]
Thanks for the suggestion. 😄

My original thought was to try and pick colours from the PSReadLine palette but on reflection I think you're right: it probably makes more sense to go with the originals.

Will leave this open for a couple of days but unless anyone thinks it's a bad idea, I'll include these colours in the next release.

**Comment by nickcox** [comment]
Released in [0.3.0](https://www.powershellgallery.com/packages/pure-pwsh/0.3.0).
